doc/nrf/libraries/others/emds.rst

```diff
@@ -9,45 +9,47 @@ Emergency data storage
 All memory areas must be provided through entries containing an ID, data pointer, and data length, using the :c:func:`emds_entry_add` function and the :c:macro:`EMDS_STATIC_ENTRY_DEFINE` macro.
 Entries to be stored when the emergency data storage is triggered need their own unique IDs that are not changed after a reboot.
 
-When all entries are added, the :c:func:`emds_load` function restores the entries into the memory areas from the flash.
+When all entries are added, the :c:func:`emds_load` function restores the entries into the memory areas from the flash/RRAM.
 
-After restoring the previous data, the application must run the :c:func:`emds_prepare` function to prepare the flash area for receiving new entries.
-If the remaining empty flash area is smaller than the required data size, the flash area will be automatically erased to increase the available flash area.
+After restoring the previous data, the application must run the :c:func:`emds_prepare` function to prepare the flash/RRAM area for receiving new entries.
+If the remaining empty flash/RRAM area is smaller than the required data size, the flash/RRAM area will be automatically erased to increase the available flash/RRAM area.
 
 The storage is done in deterministic time, so it is possible to know how long it takes to store all registered entries.
 However, this is chip-dependent, so it is important to measure the time.
 Find timing values under the "Electrical specification" section for the non-volatile memory controller in the Product Specification for the relevant SoC or the SiP you are using.
-For example, for the nRF9160 SiP, see the `Electrical specification of nRF9160`_ page.
+For example, for the nRF52840 SiP, see the `nRF52840 Product Specification`_ page.
 
 The following Kconfig options can be configured:
 
 * :kconfig:option:`CONFIG_EMDS_FLASH_TIME_WRITE_ONE_WORD_US`
 * :kconfig:option:`CONFIG_EMDS_FLASH_TIME_ENTRY_OVERHEAD_US`
 * :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US`
 
-When configuring these values, consider the time for erase when doing garbage collection in NVS.
+When configuring these values, consider the time for erase when doing garbage collection in NVS using flash.
 If partial erase is not enabled or supported, the time of a complete sector erase has to be included in the :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US`.
 When partial erase is enabled and supported by the hardware, include the time it takes for the scheduler to trigger, which is depending on the time defined in :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE_MS`.
 When changing the :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US` option, it is important that the worst time is considered.
+When configuring these values using RRAM there is not a need to consider garbage collection in the same way as for flash, as it can be written to regardless of value. But the ERASEALL functionality
```

Comment:
Is this true both for NVS and ZMS?

Reply:
I believe this is true regardless of NVS or ZMS. This is functionality for the RRAM controller so this will be true regardless of what you use on top. ERASEALL can for RRAM be blocked. When using RRAM you do not need to think about garbage collection the same way as for flash memory. How ZMS and NVS is handling this on top of this is different story.